src/sentry/receivers/sentry_apps.py

```diff
@@ -75,6 +86,14 @@ def send_comment_webhooks(organization, issue, user, event, data=None):
             user_id=(user.id if user else None),
             data=data,
         )
+    if features.has("organizations:sentry-functions", organization, actor=user):
+        if user:
+            try:
+                data["user"] = serialize(user, serializer=UserSerializer())
+            except User.DoesNotExist:
```

Comment:
@vpandiarajan20 does the serializer raise a `User.DoesNotExist`? I don't know if we need this.